IFC4x3/Properties/c/CouplingLoss_1xcHAeM5z0ie3RUCZRUAl0/Documentation.md

```diff
@@ -0,0 +1 @@
+Indicates the coupling loss of the leaky coaxial cable.
```

Comment:
Do we need the leaky part?

Reply:
Leaky coax seems to be known. E.g. https://en.wikipedia.org/wiki/Leaky_feeder?wprov=sfla1

Reply:
same page i found, it says: 

> Manufacturers and cabling professionals use the term "radiating cable"

hence my suggestion